graphics/openvdb/Portfile

```diff
@@ -78,13 +77,6 @@ variant tests description {Build unit tests} {
 
 variant docs description {Build documentation} {
     depends_build-append    port:doxygen
-    post-patch {
-        reinplace "s|doc/html|share/doc/${name}/html|" \
-            ${worksrcpath}/doc/CMakeLists.txt
-        reinplace -E \
-            "s|(DESTINATION )doc( MESSAGE_NEVER)|\\1share/doc/${name}\\2|" \
-            ${worksrcpath}/doc/CMakeLists.txt
-    }
```

Comment:
OpenVDB 8.0.1 no longer hard codes the doc paths and instead uses `${CMAKE_INSTALL_DOCDIR}`, so these replacements are no longer needed.